sr_unix/ydb_env_unset.gtc

```diff
@@ -11,6 +11,16 @@
 #################################################################
 # Usage: source "YDBDIST"/ydb_env_unset # change for distribution
 # Clears environment variables set by ydb_env_set
-unset ydb_chset ydb_dir ydb_dist ydb_etrap ydb_gbldir ydb_icu_version ydb_log ydb_rel ydb_repl_instance ydb_retention ydb_routines ydb_tmp
-unset gtm_chset gtmdir gtm_dist gtm_etrap gtmgbldir gtm_icu_version gtm_log gtm_repl_instance gtm_retention gtmroutines gtm_tmp gtmver
-unalias dse gde lke gtm mumps mupip ydb yottadb 2>/dev/null
+# This uses an M program to remove $ydb_dist from LD_LIBRARY_PATH
```

Comment:
If I don't have the M environment setup (i.e. $ydb_dist defined etc.) then issuing an error seems unfriendly when the task is to simply unset a few env vars. I would prefer we do it using "unset" commands instead of going through mumps.

Reply:
That's what it did previously. But it also needs to remove $ydb_dist from LD_LIBRARY_PATH, which is better accomplished in an M program. If no YottaDB environment is defined, it just reports that because there is nothing to unset.